Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        display(HTML(details))
</pre><p>Rather than display stuff this func needs to return an iterable of the things to be displayed</p><p> </p><p>def display_incident(incident):</p><p>&nbsp; &nbsp; details = f"""</p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;h3&gt;Selected Incident: {incident['properties.title']},&lt;/h3&gt;</p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;b&gt;Incident time: &lt;/b&gt; {incident['properties.createdTimeUtc']} - </p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;b&gt;Severity: &lt;/b&gt; {incident['properties.severity']} - </p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;b&gt;Assingned to: &lt;/b&gt;{incident['properties.owner.userPrincipalName']} - </p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;b&gt;Status: &lt;/b&gt; {incident['properties.status']}</p><p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; """</p><p>&nbsp; &nbsp; new_idx = [idx.split(".")[-1] for idx in incident.index]</p><p>&nbsp; &nbsp; incident.set_axis(new_idx, inplace=True)</p><p>&nbsp; &nbsp; return HTML(details), pd.DataFrame(incident)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>